invenio_records_resources/services/records/schema.py

```diff
@@ -42,6 +42,14 @@ def clean(self, data, **kwargs):
         return data
 
 
+class BaseGhostSchema(Schema):
+    """Base ghost schema."""
+
+    SYSTEM_USER_ID = "-1"
```

Comment:
Not sure about putting it here. The best would be to place it in the [SystemUserSchema](https://github.com/inveniosoftware/invenio-users-resources/pull/74/files#diff-503368b315245f66e0a168f6ce6a972b3e32f4258b1b6d3980b36bd3730b8941R98), but then this constant can't be used in `invenio-records-resources` as it doesn't import `invenio-users-resources` at all (it happens the other way around and created the loop import)